src/lib/tracing/init-tracing.ts

```diff
@@ -0,0 +1,54 @@
+import {DiagLogLevel, diag} from '@opentelemetry/api';
+import {OTLPTraceExporter} from '@opentelemetry/exporter-trace-otlp-http';
+import {JaegerPropagator} from '@opentelemetry/propagator-jaeger';
+import {NodeSDK, core, resources, tracing} from '@opentelemetry/sdk-node';
+import {ATTR_SERVICE_NAME, ATTR_SERVICE_VERSION} from '@opentelemetry/semantic-conventions';
+import type {AppConfig} from '../../types';
+import {DEFAULT_COLLECTOR_HOST} from '../consts';
+import {createNodekitDiagLogger} from './nodekit-diag-logger';
+import type {NodeKitLogger} from '../logging';
+
+const textMapPropagator = new core.CompositePropagator({
+    propagators: [
+        new core.W3CTraceContextPropagator(),
+        new core.W3CBaggagePropagator(),
+        new JaegerPropagator(),
```

Comment:
I think we need a way to disable `JaegerPropagator`. It may be disabled by default in the future.